components/forms/business-form/business-form.tsx

```diff
@@ -30,8 +38,41 @@ export const BusinessForm: React.FC<BusinessFormProps> = ({
     hours: initialData?.hours || "",
     email: initialData?.email || "",
     type: initialData?.type || "",
+    country: initialData?.country || "",
+    zipCode: initialData?.zipCode || "",
+    city: initialData?.city || "",
+    state: initialData?.state || "",
+    latitude: initialData?.latitude || null,
+    longitude: initialData?.longitude || null,
   });
 
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (!isLoaded || loadError) return;
```

Comment:
i would maybe just split these and if there is a loadError we can just log the loadError in the console

Reply:
I've split them with the latest commit, is that what you were looking for?

Reply:
Yes, perfect